.travis.yml

```diff
@@ -8,6 +8,13 @@ notifications:
       - "irc.freenode.org#2600hz-dev"
 
 otp_release:
+  - 17.5
+  - 17.4
+  - 17.3
+  - 17.1
+  - 17.0
+  - R16B02
+  - R16B01
```

Comment:
why are we adding all these ?
i would leave one per major release , latest of 15, 16, 17